src/helpers/articles/readTimeForArticle.js

```diff
@@ -0,0 +1,9 @@
+const readTime = (body, title, description) => {
```

Comment:
Time to read should only take into account the body of the article

Reply:
Yeah, That's right. Thank you

Reply:
> Please add tests for these feature

This feature is being tested automatically because it is part of response.
Here is the coverage report.

<img width="730" alt="Screen Shot 2019-08-26 at 18 50 05" src="https://user-images.githubusercontent.com/39116572/63707521-616d8c00-c832-11e9-81b0-4a52b34b2164.png">

Reply:
Done!

Reply:
Being hit by coverage doesn't mean the functionality has been tested, it only means that the code was run during the tests. I would expect to see an assertion that the response contains the read time property and that the read team is what is expected

Reply:
Done